From bb607f5183dd812ccd3645cf6ca8f5dfad2cb7cd Mon Sep 17 00:00:00 2001
From: packit-public-repos-bot
 <125959684+packit-public-repos-bot@users.noreply.github.com>
Date: Wed, 11 Dec 2024 11:57:20 +0100
Subject: [PATCH] Fix configuration for Packit 1.0.0

This commit fixes the configuration for the forthcoming Packit 1.0.0.
See [our blog post](https://packit.dev/posts/packit_1_0_0_action_required) for more details.
  - Job type `build` has been changed to `copr_build`.
  - Job type `production_build` has been changed to `upstream_koji_build`.
  - Key `upstream_project_name` has been changed to `upstream_package_name`.
  - Key `synced_files` has been changed to `files_to_sync`.

Please review and merge me before January 2025 otherwise packit-service jobs will fail because of an invalid configuration.

rhel-only: ci

Resolves: RHEL-70884
---
 .packit.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.packit.yml b/.packit.yml
index 9697a0df84..17bc9d1db8 100644
--- a/.packit.yml
+++ b/.packit.yml
@@ -4,7 +4,7 @@
 # Docs: https://packit.dev/docs/
 
 specfile_path: .packit_rpm/systemd.spec
-synced_files:
+files_to_sync:
   - .packit.yaml
   - src: .packit_rpm/systemd.spec
     dest: systemd.spec
